ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/address-locking.yml

```diff
@@ -22,19 +22,20 @@ ethernet_interfaces:
     description: Address Locking Interface Testing 1
     address_locking:
       ipv4: true
-
+    mac_timestamp: header
```

Comment:
why is it in address-locking file? please do it in the ethernet interface one

Reply:
Fixed.